migrations/1618843631500-users.js

```diff
@@ -0,0 +1,90 @@
+var db = require('./db')
+
+exports.up = function (next) {
+  var sql = [
+    `CREATE TYPE role AS ENUM('user', 'superuser')`,
+    `CREATE TABLE users (
+      id UUID PRIMARY KEY,
+      username VARCHAR(50) NOT NULL UNIQUE,
+      password VARCHAR(100),
+      role role NOT NULL DEFAULT 'user',
+      enabled BOOLEAN DEFAULT true,
+      twofa_code VARCHAR(100),
+      temp_twofa_code VARCHAR(100),
+      created TIMESTAMPTZ NOT NULL DEFAULT now(),
+      last_accessed TIMESTAMPTZ NOT NULL DEFAULT now(),
+      last_accessed_from TEXT,
+      last_accessed_address INET )`,
+    `CREATE TABLE "user_sessions" (
+      "sid" VARCHAR NOT NULL COLLATE "default",
+      "sess" JSON NOT NULL,
+      "expire" TIMESTAMP(6) NOT NULL )
```

Comment:
The `connect-pg-simple` library suggests using `TIMESTAMP(6)` on its initial setup. However, I experimented this change and, for now, nothing seems to be affected, so I guess this can be changed to `TIMESTAMPTZ`